pymemcache/test/utils.py

```diff
@@ -39,6 +39,14 @@ def __init__(
 
         self._contents = {}
 
+        def _serializer(key, value):
```

Comment:
Maybe `LegacyWrappingSerde._default_serializer` should convert string to bytes instead of this hack. On real client, this conversion happens on memcache write.